adapters/taboola/taboola.go

```diff
@@ -37,7 +40,7 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 
 	requestData := &adapters.RequestData{
 		Method: "POST",
-		Uri:    a.endpoint + "/" + taboolaRequest.Site.ID,
+		Uri:    a.endpoint + "/" + url.QueryEscape(a.externalUrl) + "/" + taboolaRequest.Site.ID,
```

Comment:
Appending the `externalURL` as part of the endpoint's path looks a little strange to me. Can I ask what's the usecase for this? Could it rather be sent in the request data? Also, do you do any black/white listing based on this external URL on the path? If so, does a Prebid Server host company needs to register with you before they can send requests to you?

Reply:
@mansinahar 
I agree it is a bit strange to the whole url, I changed the logic - to pass the host part only. 
And no we don't whitelist Prebid host companies based on this, it's only for monitoring the flow and for BI purposes.

Reply:
will really appreciate if you could review this PR together with this current one:
https://github.com/prebid/prebid-server/pull/2550